src/Collector/Collector.php

```diff
@@ -21,34 +21,26 @@
  */
 class Collector extends DataCollector
 {
-    /**
-     * @var Stack|null
-     */
-    private $activeStack;
-
-    /**
-     * @var int|null
-     */
-    private $capturedBodyLength;
+    private ?Stack $activeStack = null;
 
-    public function __construct(?int $capturedBodyLength = null)
+    public function __construct(private ?int $capturedBodyLength = null)
     {
         $this->reset();
-        $this->capturedBodyLength = $capturedBodyLength;
     }
 
-    /**
-     * {@inheritdoc}
-     */
+    public function __wakeup(): void
+    {
+        $this->capturedBodyLength = null;
+
+        parent::__wakeup();
+    }
+
```

Comment:
Why was this added? Did fixer do it?

Reply:
Rector inlined property to use constructor property promotion.
But after that, tests that trigger opening profiler result (which unserializes serialized before collector) failed with error "trying to get uninitialized property".

I can ddop using CPP here or leave a logic to initiale a property in wakeup